cmd/zed/dev/vng/command.go

```diff
@@ -0,0 +1,144 @@
+package vng
+
+import (
+	"bufio"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/cli/outputflags"
+	"github.com/brimdata/zed/cmd/zed/dev"
+	"github.com/brimdata/zed/cmd/zed/root"
+	"github.com/brimdata/zed/pkg/charm"
+	"github.com/brimdata/zed/pkg/storage"
+	"github.com/brimdata/zed/vng"
+	"github.com/brimdata/zed/zio"
+	"github.com/brimdata/zed/zio/zngio"
+	"github.com/brimdata/zed/zson"
+)
+
+var Cmd = &charm.Spec{
+	Name:  "vng",
+	Usage: "vng uri",
+	Short: "dump vng meta data",
+	Long: `
+vng decodes an input uri and emits the metadata sections in the format desired.`,
```

Comment:
Maybe shorten this?
```suggestion
The vng command prints a file's metadata sections.`,
```

Reply:
Input doesn't need to be a file and it doesn't "print" since it serializes in the output format desired.  Maybe there's a better way to say this.

Reply:
I don't think of -z as printing.

Reply:
Since this is ubiquitous in all of our commands, maybe we should make an issue to come up with generic language for this and update all the help text.

Reply:
Let's do that.

Reply:
> Let's do that.

#4986